src/python/grpcio/grpc/_adapter/_types.py

```diff
@@ -321,6 +354,28 @@ def create_call(self, completion_queue, method, host, deadline=float('+inf')):
     """
     return None
 
+  @abc.abstractmethod
+  def connectivity_state(self, try_to_connect=False):
```

Comment:
W.r.t. separate methods: I was mirroring the C API. Would you prefer a departure?